### DISCUSSION TITLE
Is it possible to disable the progress bar on validation requests

### DISCUSSION THREAD

**Discussion by sturatcliffe** [unpoly/unpoly#475] [General]
Hi,

As per the title really - does anyone know of an easy way to disable the progress bar, but only for validation requests?

Thanks in advance!

**Comment by adam12** [comment]
Are you on Unpoly 3? If so, you can try [forcing the request to the background](https://unpoly.com/loading-indicators#controlling-when-the-progress-bar-appears), but I thought validation requests were already in the background.

**Comment by sturatcliffe** [comment]
I'm not, we're still on V2 currently :(

**Comment by triskweline** [comment]
> I'm not, we're still on V2 currently :(

There is no way to control the progress bar in Unpoly 2. It shows up for any request that's taking long to load.

> Are you on Unpoly 3? If so, you can try [forcing the request to the background](https://unpoly.com/loading-indicators#controlling-when-the-progress-bar-appears), but I thought validation requests were already in the background.

In Unpoly 3 validation requests are in the foreground. I need to think whether being background requests would be a saner default. Probably yes when [validating for errors](https://unpoly.com/validation#validating-after-changing-a-field) but not when [updating dependent fields](https://unpoly.com/dependent-fields).

Anyways you can force validation requests into the background in Unpoly 3 like this:

```js
up.on('up:form:validate', ({ renderOptions }) => renderOptions.background = true)
```

**Reply by sturatcliffe** [reply]
Thanks @triskweline, most of our usage is very simple, i.e. setting ```up-target``` / ```up-submit``` / ```up-follow``` etc with nothing else required, so I'll look into migrating to V3!
